cord19-explorer/src/main/java/edu/asu/diging/cord19/explorer/core/model/impl/PropertiesImpl.java

```diff
@@ -0,0 +1,47 @@
+package edu.asu.diging.cord19.explorer.core.model.impl;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.annotation.Id;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+
+
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class PropertiesImpl {
```

Comment:
properties of what? what is this class used for?

Reply:
Its used to deal with the country properties

Reply:
It needs java doc and I would rename it to CountryProperties so that it does not get confused with general properties.